### PR TITLE
DNM - add back rebind, WSAStartup

### PR DESCRIPTION
**Pull request by roxanneskelly**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19994/HighFidelity-Beta-PR15773-49c5c86.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19994/HighFidelity-Beta-Interface-PR15773-49c5c86.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19994/HighFidelity-Beta-PR15773-49c5c86.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19994/HighFidelity-Beta-Interface-PR15773-49c5c86.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19994/HighFidelity-Beta-PR15773-49c5c86.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19994/HighFidelity-Quest-Beta-PR15773-49c5c86.apk)

File: libraries/networking/src/udt/Socket.cpp [hunk 1]
```diff
@@ -76,6 +76,10 @@ void Socket::rebind() {
 
 void Socket::rebind(quint16 localPort) {
     _udpSocket.close();
+#ifdef WIN32
+    WSAData ws;
+    WSAStartup(MAKEWORD(2, 2), &ws);
```
Comment (howard-stearns) [inline libraries/networking/src/udt/Socket.cpp]: - What about WSACleanup, or is that done by close?
- Is WSAStartup done by bind? If so, is there harm in doing two startups with no cleanup?
- Should we check for and report errors?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20017/HighFidelity-Beta-PR15773-df063a4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20017/HighFidelity-Beta-Interface-PR15773-df063a4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20017/HighFidelity-Beta-PR15773-df063a4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20017/HighFidelity-Beta-Interface-PR15773-df063a4.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20017/HighFidelity-Beta-PR15773-df063a4.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20017/HighFidelity-Quest-Beta-PR15773-df063a4.apk)

**Pull request closed without merging**